modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeGetQuantityMethodCheck.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeGetQuantityMethodCheck
+	extends BaseUpgradeMatcherReplacementCheck {
+
+	@Override
+	protected String formatMatcherIteration(
+		String content, String newContent, Matcher matcher) {
+
+		Pattern variableNamePattern = Pattern.compile(
+			"(\\w+)\\s+" + matcher.group(2) + "\\s*[;=),:]");
+
+		Matcher variableNameMatcher = variableNamePattern.matcher(content);
+
+		if (variableNameMatcher.find()) {
+			String variableName = variableNameMatcher.group(1);
+
+			if (!variableName.contains("CommerceOrderItem")) {
+				return newContent;
+			}
+		}
+
+		String methodCall = matcher.group();
```

Comment:
@TamyrisTorres, if you use the `JavaSourceUtil.getMethodCall(newContent, matcher.start());` you will get the complete call and call simplify the regex.